py-rse/project.Rmd

```diff
@@ -156,70 +146,66 @@ Other information may be included as sections in these files or put into files o
 -   `GOVERNANCE`
     explains how the project is run (Section \@ref(teams-martha)).
     It is still uncommon for this to be in a file of its own---it is more often included
-    in `README` or `CONTRIBUTING`---but the open source and open science communities have learned the hard way
-    that *not* being explicit about who has a voice in which decisions
-    and how contributors can tell when a decision has been made
+    in `README` or `CONTRIBUTING`---but open communities have learned the hard way
+    that *not* being explicit about who has a voice in decisions
+    and how contributors can tell what decisions have been made
     causes trouble sooner or later.
-    FIXME connect with material in teams
 
-Including four core files may seem like a lot,
-but two of these (`LICENSE` and `CONDUCT`) are usually chosen rather than written (Section \@ref(teams-license))
-and the others (`README` and `CITATION`) can be quite short to start with.
 Having these files helps new contributors orient themselves,
 and also signals that the project is well run.
-
+Creating them may seem like a lot of work,
+but `README` and `CITATION` can be quite short to start with,
+while `LICENSE` and `CONDUCT` are usually chosen rather than written.
 What's more,
-much of this content is [boilerplate][boilerplate],
+the content of these two files is usually [boilerplate][boilerplate],
 i.e.,
-the files are copied from one project to another without any changes.
+they are copied from one project to another without any changes.
 Once someone knows that a project is using the MIT License (Section \@ref(teams-license))
 and the Contributor Covenant (Section \@ref(teams-coc)),
 they don't need to read the files to know what's there.
 
-Of the four standard files generally present in most projects,
-we have three already present in our Zipf's Law project.
-`README.md` was created as a part of Chapter \@ref(git-cmdline)
-`CONDUCT.md` and `LICENSE.md` were both created in Chapter \@ref(teams).
-At this point in our project development,
-it doesn't make sense to create separate files for the other standard files.
-However, you may choose to include subheadings in the main directory's `README.md`
-with information about contributing to the project
-Eventually,
-you may also include governance processes,
-citation details,
-and contributors.
-
-## How should I structure the rest of my project? {#project-organize}
+Three of these files are already present in our Zipf's Law project:
+`README.md` was created in Chapter \@ref(git-cmdline),
+while `CONDUCT.md` and `LICENSE.md` were added in Chapter \@ref(teams).
```

Comment:
In the Great Rearrangement this project chapter/content will possibly come before the teams chapter (see the following comment from #415) 
https://github.com/merely-useful/merely-useful.github.io/pull/415#issuecomment-653984887